### ISSUE TITLE
Import hint for Transform3D/Position3D/Node3D when importing from blender

### ISSUE THREAD

**Issue by elvisish** [godotengine/godot-proposals#6040]
### Describe the project you are working on

First person shooter.

### Describe the problem or limitation you are having in your project

Importing objects from blender, there aren't any options for a simple position node, instead it can only create meshes and collisions.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

It would be convenient if a suffix could be added to a blender object that imported it as a basic Transform/Node that could be used for simple positioning.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

`Plain axis` could be repurposed for it, since `single arrow` is used for raycasts:
![image](https://user-images.githubusercontent.com/16231628/210511524-6d89c7ec-b7c1-47e1-8a1e-5ec1c47548ff.png)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

I think it could be done with a tool script, but it would be inconvenient for most users.

### Is there a reason why this should be core and not an add-on in the asset library?

It would be far more convenient and very useful for the majority of users importing from Blender.

**Comment by Calinou** [comment]
Shouldn't these be imported as Marker3D nodes instead?

**Comment by elvisish** [comment]
> Sounds good, but shouldn't these be imported as Marker3D nodes instead (Position3D in 3.x)?

Yeah, I actually meant to edit this yesterday after I remembered it existed, Marker3D would probably be the best node.
